pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -63,6 +64,8 @@ const (
 
 	// MachineInstanceTypeLabelName as annotation name for a machine instance type
 	MachineInstanceTypeLabelName = "machine.openshift.io/instance-type"
+
+	MachineSetLabelName = "machine.openshift.io/cluster-api-machineset"
```

Comment:
How does this relate to the upstream design?

Reply:
Upstream does something similar, they use machine deployment and cluster name for availability set.